### ISSUE TITLE
Cannot run Dockerfile: The command '/bin/sh -c apt-get update' returned a non-zero code: 100

### ISSUE THREAD

**Issue by zhangysh1995** [panda-re/lava#34]
````
[zhangys@chcpu12a docker]$ sudo docker build .
[sudo] password for zhangys:
Sending build context to Docker daemon  10.24kB
Step 1/53 : FROM i386/debian:stretch
stretch: Pulling from i386/debian
bd6445c9478b: Pull complete
Digest: sha256:c576bf44166305ac3ccfc499a97d8b24d9e6016967bc377d7d5ee0518dd0b50c
Status: Downloaded newer image for i386/debian:stretch
 ---> 249c718c28b7
Step 2/53 : RUN echo deb http://httpredir.debian.org/debian wheezy-backports main >> /etc/apt/sources.list
 ---> Running in e7885427ca0a
 ---> 943de2a63b21
Removing intermediate container e7885427ca0a
Step 3/53 : RUN apt-get update
 ---> Running in 574bb636d35e
Ign:1 http://cdn-fastly.deb.debian.org/debian wheezy-backports InRelease
Get:2 http://security-cdn.debian.org/debian-security stretch/updates InRelease [93.6 kB]
Ign:3 http://cdn-fastly.deb.debian.org/debian stretch InRelease
Get:4 http://cdn-fastly.deb.debian.org/debian stretch-updates InRelease [90.3 kB]
Get:7 http://security-cdn.debian.org/debian-security stretch/updates/main i386 Packages [501 kB]
Ign:5 http://cdn-fastly.deb.debian.org/debian wheezy-backports Release
Get:6 http://cdn-fastly.deb.debian.org/debian stretch Release [118 kB]
Get:10 http://cdn-fastly.deb.debian.org/debian stretch-updates/main i386 Packages [27.2 kB]
Ign:8 http://httpredir.debian.org/debian wheezy-backports/main i386 Packages
Ign:8 http://cdn-fastly.deb.debian.org/debian wheezy-backports/main i386 Packages
Get:11 http://cdn-fastly.deb.debian.org/debian stretch Release.gpg [2434 B]
Ign:9 http://httpredir.debian.org/debian wheezy-backports/main all Packages
Ign:9 http://cdn-fastly.deb.debian.org/debian wheezy-backports/main all Packages
Ign:8 http://httpredir.debian.org/debian wheezy-backports/main i386 Packages
Ign:8 http://cdn-fastly.deb.debian.org/debian wheezy-backports/main i386 Packages
Get:12 http://cdn-fastly.deb.debian.org/debian stretch/main i386 Packages [7062 kB]
Ign:9 http://httpredir.debian.org/debian wheezy-backports/main all Packages
Ign:9 http://cdn-fastly.deb.debian.org/debian wheezy-backports/main all Packages
Ign:8 http://httpredir.debian.org/debian wheezy-backports/main i386 Packages
Ign:8 http://cdn-fastly.deb.debian.org/debian wheezy-backports/main i386 Packages
Ign:9 http://httpredir.debian.org/debian wheezy-backports/main all Packages
Ign:9 http://cdn-fastly.deb.debian.org/debian wheezy-backports/main all Packages
Ign:8 http://httpredir.debian.org/debian wheezy-backports/main i386 Packages
Ign:8 http://cdn-fastly.deb.debian.org/debian wheezy-backports/main i386 Packages
Ign:9 http://httpredir.debian.org/debian wheezy-backports/main all Packages
Ign:9 http://cdn-fastly.deb.debian.org/debian wheezy-backports/main all Packages
Ign:8 http://httpredir.debian.org/debian wheezy-backports/main i386 Packages
Ign:8 http://cdn-fastly.deb.debian.org/debian wheezy-backports/main i386 Packages
Ign:9 http://httpredir.debian.org/debian wheezy-backports/main all Packages
Ign:9 http://cdn-fastly.deb.debian.org/debian wheezy-backports/main all Packages
Err:8 http://httpredir.debian.org/debian wheezy-backports/main i386 Packages
  404  Not Found
Ign:8 http://cdn-fastly.deb.debian.org/debian wheezy-backports/main i386 Packages
Ign:9 http://cdn-fastly.deb.debian.org/debian wheezy-backports/main all Packages
Fetched 7894 kB in 15s (517 kB/s)
Reading package lists...
W: The repository 'http://httpredir.debian.org/debian wheezy-backports Release' does not have a Release file.
E: Failed to fetch http://httpredir.debian.org/debian/dists/wheezy-backports/main/binary-i386/Packages  404  Not Found
E: Some index files failed to download. They have been ignored, or old ones used instead.
The command '/bin/sh -c apt-get update' returned a non-zero code: 100


````

**Comment by AndrewFasano** [comment]
I believe this issue was fixed in 4ea27fd184ddd80a6a0353d862e421e9eaeab19f. Are you using the latest version of LAVA?

**Comment by zhangysh1995** [comment]
> I believe this issue was fixed in [4ea27fd](https://github.com/panda-re/lava/commit/4ea27fd184ddd80a6a0353d862e421e9eaeab19f). Are you using the latest version of LAVA?

I think I downloaded the source code from `release` page. It seems it is not updated. I use the `master` version and it could build now.
